src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,6 +16,9 @@
 #include <string.h>
 #include <sys/stat.h>
 #include "coreruncommon.h"
+#if defined(__LINUX__)
```

Comment:
I don't see need for it. I prefer to reduce `__OS__` as much as possible - only for code parts that are really compiling and working under `OS`. It may be valid later for android or some other system.

But it's up to .Net maintainers.